modules/apps/frontend-js/frontend-js-loader-modules-extender/src/main/java/com/liferay/frontend/js/loader/modules/extender/internal/npm/builtin/BaseBuiltInJSModuleServlet.java

```diff
@@ -159,24 +159,26 @@ private void _sendResource(
 		if (inputStream == null) {
 			String moduleName = resourceDescriptor.getPackagePath();
 
-			if (extension.equals("map")) {
-				JSModule jsModule = jsPackage.getJSModule(
-					moduleName.substring(0, moduleName.length() - 7));
+			if (moduleName != null) {
+				if (extension.equals("map")) {
+					JSModule jsModule = jsPackage.getJSModule(
+						moduleName.substring(0, moduleName.length() - 7));
 
-				if (jsModule != null) {
-					inputStream = jsModule.getSourceMapInputStream();
-				}
-			}
-			else {
-				if (extension.equals("js")) {
-					moduleName = moduleName.substring(
-						0, moduleName.length() - 3);
+					if (jsModule != null) {
+						inputStream = jsModule.getSourceMapInputStream();
+					}
 				}
+				else {
+					if (extension.equals("js")) {
+						moduleName = moduleName.substring(
+							0, moduleName.length() - 3);
+					}
 
-				JSModule jsModule = jsPackage.getJSModule(moduleName);
+					JSModule jsModule = jsPackage.getJSModule(moduleName);
 
-				if (jsModule != null) {
-					inputStream = jsModule.getInputStream();
+					if (jsModule != null) {
+						inputStream = jsModule.getInputStream();
+					}
```

Comment:
Please log the fact this is null so that we make it possible to diagnose potential issues in the future